mbtools/copy_resources_s3.py

```diff
@@ -122,18 +134,28 @@ def main():
                         help='bucket name')
     parser.add_argument('s3_prefix', type=str,
                         help='prefix for s3 files')
+    parser.add_argument('-o', '--csv', const=None,
+                        help='hash to filename csv output file')
+
     args = parser.parse_args()
 
     new_resource_dir = Path(args.new_resource_path).resolve(strict=True)
 
     metadata_file = Path(args.metadata_file)
 
+    csv_file = None
+    if args.csv is not None:
+        csv_file = Path(args.csv)
+        if not csv_file.exists():
+            csv_file.parent.mkdir(parents=True, exist_ok=True)
+            csv_file.write_text("[]")
```

Comment:
Minor: This line can be removed since you open the file later in `'w'` mode it will get truncated + this would be invalid CSV anyways.

Reply:
Do you mean all 4 of these lines or just the `write_text([])` function? I assume you mean all of it.

Reply:
Oh, I meant just the `csv_file.write_text("[]")`. I think the rest may be required if the input CSV path includes a non-existing directory (otherwise it will probably error when trying to open the file later)